sagemathinc/cocalc
# x11/keyboard: support variants

# Description
yesterday keyboard layout, today variants.

# Testing Steps
1. selecting "English (us)" should not pre-select a variant, but e.g. now you can select Cherokee!
2. selecting "German" or "Austria" should pre-select "nodeadkeys", because that's the usual variation.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
